### DISCUSSION TITLE
Valid JSON feed creates 500 Server Error when importing into Google Sheets Importer

### DISCUSSION THREAD

**Discussion by fritzisa** [code4recovery/12-step-meeting-list#1492] [Q&A]
I started with a District's Google Sheet, which JSON Feed Validator validates:
https://docs.google.com/spreadsheets/d/1AEjqCJqoHYTLPtg7pElBTDsWSWKC_Uy-dYp0qTD6qS4/pubhtml

When I import that into Google Sheets Importer, I get a 500 | Server Error

Any assistance is greatly appreciated.

**Comment by kiyote33** [comment]
Hi @fritzisa, 
We have a process on our [code4recovery.org](https://sheets.code4recovery.org/) website for converting a Google Spreadsheet into a linked JSON file. I ran your linked Google spreadsheet through that process and was able to convert the Google Spreadsheet in your link into a JSON file at this link: [https://[sheets.code4recovery.org/storage/12Ga8uwMG4WJ8pZ_SEU7vNETp_aQZ-2yNVsYDFqIwHyE.json](https://sheets.code4recovery.org/storage/12Ga8uwMG4WJ8pZ_SEU7vNETp_aQZ-2yNVsYDFqIwHyE.json)](https://[sheets.code4recovery.org/storage/12Ga8uwMG4WJ8pZ_SEU7vNETp_aQZ-2yNVsYDFqIwHyE.json](https://sheets.code4recovery.org/storage/12Ga8uwMG4WJ8pZ_SEU7vNETp_aQZ-2yNVsYDFqIwHyE.json)).  I then imported that JSON file into my local test site, successfully loading all 676 meetings for District 48. Let me know if this works for you or you need a bit more help.

**Comment by joshreisner** [comment]
hi @fritzisa i dont think the validator message is accurate - if you look at the data it's all empty

<img width="1367" alt="Screenshot 2024-08-20 at 7 37 39 PM" src="https://github.com/user-attachments/assets/1a33c5a7-eceb-48d4-948d-2f060aec1b02">

i would remove the first row that says "meetings" ~~and then enter the sheet URL that you use when editing (https://docs.google.com/spreadsheets/d/1AEjqCJqoHYTLPtg7pElBTDsWSWKC_Uy-dYp0qTD6qS4/edit?gid=1764299868#gid=1764299868),~~ and i think it will work

**Reply by joshreisner** [reply]
actually never mind about using the edit url, this url is fine as well! just remove that first row and you should be good

i've added some column validation so it should tell you now it's missing a required column rather than showing a 500

**Comment by fritzisa** [comment]
Thanks so much for your help!

Scott Fritzinger  |  Area 59 Web Servant  |  (610) 762-7792
